robyn/scaffold/structured/sqlalchemy/adaptors/models/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .user import Base as user
+
+metadata = [user.metadata]
```

Comment:
is this a standard pattern?

Reply:
yes, https://alembic.sqlalchemy.org/en/latest/autogenerate.html

alembic expects the model's sqlalchemy metadata to be passed in env.py
having it in models/__init__.py for better structure

which is used in alembic/env.py like so
```python
from adaptors import models
target_metadata = models.metadata
```